clap_complete/tests/snapshots/home/static/exhaustive/fish/fish/completions/exhaustive.fish

```diff
@@ -55,11 +59,15 @@ complete -c exhaustive -n "__fish_exhaustive_using_subcommand global; and __fish
 complete -c exhaustive -n "__fish_exhaustive_using_subcommand global; and __fish_seen_subcommand_from help" -f -a "two"
 complete -c exhaustive -n "__fish_exhaustive_using_subcommand global; and __fish_seen_subcommand_from help" -f -a "help" -d 'Print this message or the help of the given subcommand(s)'
 complete -c exhaustive -n "__fish_exhaustive_using_subcommand action" -l set -d 'value' -r
-complete -c exhaustive -n "__fish_exhaustive_using_subcommand action" -l choice -d 'enum' -r -f -a "{first\t'',second\t''}"
+complete -c exhaustive -n "__fish_exhaustive_using_subcommand action" -l choice -d 'enum' -r -f -a "first\t''
+second\t''"
 complete -c exhaustive -n "__fish_exhaustive_using_subcommand action" -l set-true -d 'bool'
 complete -c exhaustive -n "__fish_exhaustive_using_subcommand action" -l count -d 'number'
 complete -c exhaustive -n "__fish_exhaustive_using_subcommand action" -s h -l help -d 'Print help'
-complete -c exhaustive -n "__fish_exhaustive_using_subcommand quote; and not __fish_seen_subcommand_from cmd-single-quotes cmd-double-quotes cmd-backticks cmd-backslash cmd-brackets cmd-expansions escape-help help" -l choice -r -f -a "{another shell\t'something with a space',bash\t'bash (shell)',fish\t'fish shell',zsh\t'zsh shell'}"
+complete -c exhaustive -n "__fish_exhaustive_using_subcommand quote; and not __fish_seen_subcommand_from cmd-single-quotes cmd-double-quotes cmd-backticks cmd-backslash cmd-brackets cmd-expansions escape-help help" -l choice -r -f -a "another shell\t'something with a space'
+bash\t'bash (shell)'
+fish\t'fish shell'
+zsh\t'zsh shell'"
```

Comment:
What happens when there is a newline in the description?

Reply:
I tested how Fish handles newlines in descriptions, and here are my findings:

```fish
$ complete -c ls -l flag -r -f -a "a\t'This is A,
which is a description'
b\t'This is B,\nwhich is a description'"
$ ls --flag <TAB>
a  (This is A, which is a description)  b  (This is B,\nwhich is a description)
```

It seems that normal newlines `\n` are treated as spaces, which works fine.
However, when `\n` appears as a literal in the description, it is displayed as \n in the completion list.

Would this be an issue?
If a fix is needed, I’m happy to make the necessary changes.

Reply:
Oh right, its quoted.